charts/tfy-gpu-operator/Chart.yaml

```diff
@@ -1,22 +1,27 @@
 apiVersion: v2
 name: tfy-gpu-operator
-version: 0.1.9
+version: 0.1.10.rc.1
 description: "Truefoundry GPU Operator"
 maintainers:
   - name: truefoundry
 dependencies:
   - condition: clusterType.awsEks
     name: gpu-operator
     repository: https://helm.ngc.nvidia.com/nvidia
-    version: v23.3.1
+    version: v23.9.0
     alias: aws-eks-gpu-operator
   - condition: clusterType.gcpGkeStandard
     name: dcgm-exporter
     repository: https://nvidia.github.io/dcgm-exporter/helm-charts
-    version: 3.1.2
+    version: 3.1.7
     alias: gcp-gke-standard-dcgm-exporter
+  - condition: clusterType.azureAks
```

Comment:
Should we use gpu-operator - https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/23.9.0/microsoft-aks.html . That feels like the recommended option

Reply:
Yeah, not a bad idea, will try it out